### PR TITLE
Merge develop into master

### PR DESCRIPTION
**Pull request by clausmullie**
Merge develop into master

### CODE REVIEW

File: CODE_OF_CONDUCT.md
Comment (Hugo-ter-Doest) [inline CODE_OF_CONDUCT.md]: officialproject --> two words

File: GOVERNANCE.md
Comment (Hugo-ter-Doest) [inline GOVERNANCE.md]: technical merit: functional merit as well.

File: GOVERNANCE.md
Comment (Hugo-ter-Doest) [inline GOVERNANCE.md]: architecture

File: GOVERNANCE.md
Comment (Hugo-ter-Doest) [inline GOVERNANCE.md]: review

File: GOVERNANCE.md
Comment (Hugo-ter-Doest) [inline GOVERNANCE.md:69]: what is meant by super majority? 2/3?

File: GOVERNANCE.md
Comment (Hugo-ter-Doest) [inline GOVERNANCE.md:78]: This calls for max. one member per party

File: GOVERNANCE.md
Comment (Hugo-ter-Doest) [inline GOVERNANCE.md]: I think super majority, or even unanimous

File: how-to-respond-nl.md
Comment (Hugo-ter-Doest) [inline how-to-respond-nl.md]: doorontwikkeld

File: how-to-respond-nl.md
Comment (Hugo-ter-Doest) [inline how-to-respond-nl.md]: wisseling tussen u/uw jij/jullie, ook bij volgende bullet

File: how-to-respond-nl.md
Comment (Hugo-ter-Doest) [inline how-to-respond-nl.md]: Een deelname -> Deelname (zonder lidwoord)

File: introduction-nl.md
Comment (Hugo-ter-Doest) [inline introduction-nl.md]: collaboratieve -> samenwerkende

Comment (Hugo-ter-Doest) [inline introduction-nl.md]: -ondersteuning: streepje weglaten

File: introduction-nl.md
Comment (Hugo-ter-Doest) [inline introduction-nl.md]: herzien: lezen

File: introduction-nl.md
Comment (Hugo-ter-Doest) [inline introduction-nl.md]: worden ontvangen door -> binnen zijn bij

File: questions-en.md
Comment (Hugo-ter-Doest) [inline questions-en.md]: ranges from to

File: questions-nl.md
Comment (Hugo-ter-Doest) [inline questions-nl.md]: -ondersteuning -> streepje weglaten

File: questions-nl.md
Comment (Hugo-ter-Doest) [inline questions-nl.md]: beetje lastig: 
governance op basis van samenwerking, waarbij ook wordt samengewerkt rond gedeelde roadmaps etc.

File: questions-nl.md
Comment (Hugo-ter-Doest) [inline questions-nl.md]: gepropageerd naar

File: questions-nl.md
Comment (Hugo-ter-Doest) [inline questions-nl.md]: eens worden over

**Review by Hugo-ter-Doest — CHANGES_REQUESTED** [review]
Prima stukken, complimenten voor dit werk!

Hier en daar taalfoutjes eruit gehaald.

**Comment by ElenaFdR** [comment]
Do not merge this until after #5 is merged.

File: questions-nl.md
Comment (ElenaFdR) [reply questions-nl.md]: Does your comment completely replace the second bullet above? Is the reference to the issue tracker not necessary? (I'll update the English version to match this one.)

File: how-to-respond-nl.md
Comment (ElenaFdR) [reply how-to-respond-nl.md]: Dit gebeurt in alle Nederlandse versies - heb jij een voorkeur?

File: GOVERNANCE.md
Comment (ElenaFdR) [reply GOVERNANCE.md:69]: I've added 2/3 for now, but the actual community/steering team could pick a higher super majority if they were more comfortable with that.

File: questions-nl.md
Comment (Hugo-ter-Doest) [reply questions-nl.md]: Nee, het stukje "(feature en technical) en open issue tracker" moet er wel bij

Comment (Hugo-ter-Doest) [reply questions-nl.md]: open issue tracker --> openbare issue tracker

File: how-to-respond-nl.md
Comment (Hugo-ter-Doest) [reply how-to-respond-nl.md]: voorkeur voor u/uw

File: GOVERNANCE.md
Comment (ElenaFdR) [reply GOVERNANCE.md]: I've changed this to supermajority for now.

File: GOVERNANCE.md
Comment (ElenaFdR) [reply GOVERNANCE.md:78]: That depends on how much you trust the parties, and how diverse the community is. Do you want to add that now already, or see how the community develops?

Comment (Ainali) [reply GOVERNANCE.md:78]: @Hugo-ter-Doest Do you want that regulated by the document? Or is it enough that everyone is aware of it and that the team collectively will make sure to keep the balance when appointing new members?

Comment (Ainali) [reply GOVERNANCE.md:78]: I created #7 to resolve this issue and unblock this merge.

File: how-to-respond-nl.md
Comment (ElenaFdR) [reply how-to-respond-nl.md]: @hugo-ter-doest, we're going to go ahead and merge #5 without making this change right now - the typo fixes are urgent (because [typos lead to decreased trust](https://www.bbc.co.uk/news/education-14130854)) while jullie/uw is only stylistic. We'll pick this up later this week.

Comment (ElenaFdR) [reply how-to-respond-nl.md]: I've logged this in #8.

File: GOVERNANCE.md
Comment (Hugo-ter-Doest) [reply GOVERNANCE.md:78]: For now it is fine to see how the community develops. If everyone is aware of the risk, we can keep the balance.

**Review by Ainali — APPROVED** [review]

File: GOVERNANCE.md
Comment (ElenaFdR) [inline GOVERNANCE.md]:
```suggestion
The default decision making process is lazy consensus. This means that any decision is considered supported by the team making it as long as no one objects. Silence on any consensus decision is implicit agreement and equivalent to explicit agreement. Explicit agreement may be stated at will.
```

File: how-to-respond-en.md
Comment (ElenaFdR) [inline how-to-respond-en.md]:
```suggestion
Dimpact is responsible for communication about this market consultation. You can contact Niels Lindeboom at <niels.lindeboom@dimpact.nl> or 06-22270923.
```

**Pull request merged**
